app/controllers/turns_controller.rb

```diff
@@ -1,13 +1,20 @@
 class TurnsController < ApplicationController
+  protect_from_forgery except: :create
+
   def create
     game = Game.find(params[:game_id])
 
     move = Move.from_direction(
       Coord.new(turn_params[:x].to_i, turn_params[:y].to_i),
       turn_params[:direction]
     )
+    binding.irb
+
+    user = User.find(params[:user_id])
+
+    game.turns.create(move: move, player: user.id)
```

Comment:
We also want to have a relational mapping between users and turns here instead of saving the user id as a string. Then you can do `game.turns.create(move: move, user: @user)`